lib/api/bucketDeleteCors.js

```diff
@@ -44,18 +47,18 @@ export default function bucketDeleteCors(authInfo, request, log, callback) {
             });
             // TODO: add 'deleteBucketCors' to Utapi Client action map
             // pushMetric('deleteBucketCors', log, { bucket: bucketName });
-            return callback();
+            return callback(null, corsHeaders);
         }
 
         log.trace('deleting cors configuration in metadata');
         bucket.setCors(null);
         return metadata.updateBucket(bucketName, bucket, log, err => {
             if (err) {
-                return callback(err);
+                return callback(err, corsHeaders);
```

Comment:
Yep. It seems strange at first but if you think about it, the cors headers are the green light for CORS requests. If a cors config currently exists, it makes sense to permit a CORS request that matches to delete the config (it hasn't been deleted yet). Once the config is deleted, the same request would no longer be permitted.